<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="loginPageButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignupButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>690</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorText">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>660</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signupPassword">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>300</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signupUsername">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>200</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signupEmail">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPassword">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstName">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>490</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>firstName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastName">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>lastName</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
